android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletActivity.java

```diff
@@ -280,12 +281,13 @@ private void addRemoveSecureFlag(boolean add) {
         }
     }
 
-    private void replaceNavigationFragments(int type, boolean doNavigate) {
+    private void replaceNavigationFragments(int type, boolean doNavigate, boolean isOnboarding) {
         mShowBiometricPrompt = true;
         if (cryptoWalletOnboardingViewPager != null && cryptoWalletOnboardingPagerAdapter != null) {
             if (type == RESTORE_WALLET_ACTION) {
                 mShowBiometricPrompt = false;
-                RestoreWalletFragment restoreWalletFragment = new RestoreWalletFragment();
+                RestoreWalletFragment restoreWalletFragment =
```

Comment:
we should not pass arguments to constructor. instead we should pass the argument to fragment.